intg-androidtv/apps.py

```diff
@@ -14,6 +15,12 @@
     "Ziggo": {"url": "ziggogo://"},
     "Videoland": {"url": "videoland-v2://"},
     "Steam Link": {"url": "steamlink://"},
+    "Waipu TV": {"url": "waipu://tv"},
+#    "Magenta TV": {"url": "https://www.telekom.de/magenta-tv"},
```

Comment:
Uncomment or remove

Reply:
Hello zehm,
Sorry I was a little bit confused last week. I desperatly tried to add more keys to the integration this is why there was garbage in tv.py. For switch apps I am used to adb. I only did one pull request do you get it now everytime I sync? I am now concentrating on getting ZDF and Magenta run lokaly before I sync again.

Ralf

Reply:
No worries. As soon as you open a PR, any future pushes to the corresponding branch will end up in the PR :-)